### DISCUSSION TITLE
Need Help EraserBrushV5.3

### DISCUSSION THREAD

**Discussion by vavilon224** [fabricjs/fabric.js#9520] [Q&A] [answered]
Need Help EraserBrushV5.3

Hello everyone, tell me how to install using npm in my factory "EraserBrush",

My React + Fabric.js 5.3.0 Stack

**Comment by vavilon224** [comment]
@melchiar @ShaMan123

**Reply by ShaMan123** [reply, marked as answer]
Though I wrote the EraserBrush and it is not availale yet in v6 #8499, I am very much rooted in v6.
I don't really remember the mumbo jumbo of the custom build

https://github.com/fabricjs/fabric.js/blob/3251c4614b8fc5fc34634d139271653b3ecd3c6e/package.json#L48

so 
```bash
node build.js modules=ALL requirejs exclude=gestures,accessors
```
should work

**Reply by ShaMan123** [reply]
I will rebase #8499 so you could work on that branch in v6

**Reply by vavilon224** [reply]
thank you very much) I'll try

**Comment by vavilon224** [comment]
@ShaMan123 So where should I call this command??))

**Reply by ShaMan123** [reply]
sorry,
add a postinstall script in `package.json`
```
"postinstall": "cd node_modules && node build.js modules=ALL requirejs exclude=gestures,accessors"
```

Maybe you will need but I don't remember
```
"postinstall": "cd node_modules && npm i && node build.js modules=ALL requirejs exclude=gestures,accessors"
```

**Reply by ShaMan123** [reply]
I think there is a flag `fast`
flag it to avoid minification

**Comment by vavilon224** [comment]
> cd node_modules/fabric && npm run build
> fabric@5.3.0 build
> node build.js modules=ALL requirejs exclude=gestures,accessors,erasing

Built distribution to dist/fabric.js (requirejs-compatible)
Minification failed using uglifyjs with uglifyjs  --compress --mangle --output fabric.min.js fabric.js
Minifier error output:
Error: Command failed: uglifyjs  --compress --mangle --output fabric.min.js fabric.js
"uglifyjs" �� ���� ����७��� ��� ���譥�
��������, �ᯮ��塞�� �ணࠬ��� ��� ������ 䠩���.

**Comment by vavilon224** [comment]
@ShaMan123 Thank you, everything worked out for me))

    "postinstall": "cd node_modules && cd fabric && node build.js modules=ALL requirejs exclude=gestures,accessors"


I'll leave it if anyone needs it
